tests/components/blastbot_cloud/test_config_flow.py

```diff
@@ -0,0 +1,70 @@
+"""Tests for the Blastbot Cloud config flow."""
+from aiohttp import ClientError
+
+from homeassistant import data_entry_flow
+from homeassistant.components.blastbot_cloud import config_flow
+from homeassistant.const import CONF_PASSWORD, CONF_USERNAME, HTTP_INTERNAL_SERVER_ERROR
+
+from tests.async_mock import patch
+
+
+async def test_show_form(hass):
+    """Test that the form is served with no input."""
+    flow = config_flow.BlastbotCloudConfigFlow()
```

Comment:
Please use the core interfaces to start the tests. Here use `hass.config_entries.flow.async_init`. Use the scaffold script to get a good base for the tests.